AuxTel/AuxTel-Troubleshooting/ATCS-Troubleshooting/ATMCS-fails-enable.rst

```diff
@@ -0,0 +1,67 @@
+.. Include one Primary Author and list of Contributors (comma separated) between the asterisks (*):
+.. |author| replace:: *I. Sotuela*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *P. Venegas*, *K. Aubel*
+
+.. This is the label that can be used as for cross referencing this procedure.
+.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
+.. _ATCS-Troubleshooting-AuxTel-Mount-Control-System-Fails-to-Enable:
+.. Each section should includes a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+.. _E-Stop-Procedure: https://obs-ops.lsst.io/AuxTel/Non-Standard-Operations/ATCS/E-Stop-Procedure.html
```

Comment:
For links to internal repository documents use internal labels :ref:`link text <label>`. this way, if the location of the file change, the link will still work .